python/chronos/use-case/pytorch-forecasting/DeepAR/deepar.py

```diff
@@ -52,7 +52,7 @@
 from pytorch_forecasting.metrics import NormalDistributionLoss
 from pytorch_forecasting.models.deepar import DeepAR
 from pytorch_forecasting.utils import profile
-from bigdl.nano.pytorch.trainer import Trainer
+from bigdl.chronos.common.pytorch import TSTrainer
```

Comment:
Might be better to use `bigd.chronos.pytorch.TSTrainer`?

Reply:
@TheaperDeng

Reply:
yeah, that could be a shorter name.